viapy/api.py

```diff
@@ -130,7 +130,11 @@ def rdf(self):
         """VIAF data for this entity as :class:`rdflib.Graph`"""
         start = time.time()
         graph = rdflib.Graph()
-        graph.parse(self.uri)
+        # 2025 update: Accept header now required, so use requests.get to retrieve RDF
+        response = requests.get(self.uri, headers={"Accept": "application/rdf+xml"})
+        if response.status_code == requests.codes.ok:
+            graph.parse(data=response.text, format="xml")
+        response.raise_for_status()
```

Comment:
should this be under an `else` ?

Reply:
I don't think so, `raise_for_status` will only do anything if the status isn't a success status. In fact you could actually call it before the `graph.parse` call and avoid the conditional altogether! Should I make that change?

Reply:
Yes, why don't you do that. Sounds simpler & cleaner.